modules/candidate_parameters/jsx/ProbandInfo.js

```diff
@@ -2,7 +2,7 @@ let ProbandInfo = React.createClass(
   {
```

Comment:
changes to this files requires a npm run compile (i dont see it being done here)

Reply:
There are no longer any compiled files tracked on the major branch.